jujugui/static/gui/src/app/assets/css/_animations.scss

```diff
@@ -7,3 +7,23 @@
         opacity: 1;
     }
 }
+
+%fade-in {
+    $fade-in-duration: .1s;
+    animation-duration: $fade-in-duration;
+    animation-name: fadeIn;
+    animation-fill-mode: forwards;
+    opacity: 0;
+}
+
+%stagger-fade-in {
+    @extend %fade-in;
+
+    @for $i from 1 through 99 {
+        $fade-in-stagger-speed: .01s;
```

Comment:
Oh, sorry I must have copied this line: https://github.com/anthonydillon/inspector-animations/blob/master/static/scss/_states.scss#L120